fr/api/rest-api-v2.md

```diff
@@ -0,0 +1,127 @@
+---
+id: rest-api-v2
+title: Rest API (v2)
+---
+
+## Aperçu
+
+Centreon Web Rest API v2 a pour objectif de remplacer la première version.
+
+Pour commencer à l'utiliser, référez-vous à la [page
+dédiée](https://docs.centreon.com/api/centreon-web/).
+
+## Créer une collection Postman depuis la définition OpenAPI
+
+Afin de manipuler l'API plus efficacement ou seulement pour comprendre comment
+elle fonctionne, il n'y a rien de plus pratique qu'une collection Postman.
+
+### Importer la définition OpenAPI
+
+Depuis l'espace de travail, cliquer sur le bouton **Import**.
+
+![image](../assets/api/postman-import.png)
+
+Aller dans l'onglet **Link** et entrer l'URL vers la définition OpenAPI de
+GitHub :
+
+```text
+https://raw.githubusercontent.com/centreon/centreon/master/doc/API/centreon-api-v2.yaml
+```
+
+![image](../assets/api/postman-import-link.png)
+
+Cliquer sur **Continue** et modifier quelques paramètres par défaut :
+
+- Sélectionner seulement *Collection* comme méthode d'import,
+- Sélectionner *Example* pour le champ *Request parameter generation* car cela
+  amène plus de contexte aux variables d'URL ou aux paramètres de requête,
+- Sélectionner *Tags* pour le champ *Folder organization* car cela permet
+  d'arranger la collection comme dans la documentation.
+
+Cliquer ensuite sur **Import** et attendre quelques secondes que le processus
+aboutisse.
+
+![image](../assets/api/postman-import-feedback.png)
+
+### Ajouter un environnement
+
+Pour pouvoir communiquer avec vote plateforme, la collection va utiliser des
+variables d'environnement pour déinir les informations d'accès ainsi que les
+identifiants.
+
+Depuis l'espace de travail, cliquer sur l'icone **Manage Environments**.
+
+![image](../assets/api/postman-environment.png)
+
+Cliquer sur **Add** et definir les variables comme suit avec les informations
+de la plateforme :
+
+![image](../assets/api/postman-environment-add.png)
```

Comment:
same comment > why {{version}} is not a variable to be defined ?

Reply:
I just added a section about it. It's more of a collection-wide variable.